modules/varnish/templates/default.vcl

```diff
@@ -321,7 +326,6 @@ sub mw_vcl_recv {
 
 	// HACK for phabricator.wikimedia.org/T217669
 	if (req.url ~ "/w(iki)?/undefined/api.php") {

```

Comment:
Why not move this to the top?

Reply:
Wouldn't doing returning the synthetic response before, prevent the backend from ever being set?